laws/src/main/scala/spire/laws/arb.scala

```diff
@@ -2,6 +2,8 @@ package spire.laws
 
 import java.math.BigInteger
 
+import spire.math.extras.{FixedPoint, FixedScale}
```

Comment:
I don't think this import is used. There are a few files with just imports added, so we can probably remove the imports for now.